### PR TITLE
Added getValidatorMessages and fix getstatus api error

### PR DESCRIPTION
**Pull request by samparsky**
Fixes issue #23 and #36

### CODE REVIEW

File: routes/channel.js [hunk 1]
```diff
@@ -43,6 +53,35 @@ function getStatus(withTree, req, res) {
 	})
 }
 
+// Implementation of get validator messages last recent N
+// validator messages
```
Comment (Ivshti) [inline routes/channel.js]: shouldn't it be "getvalidatorMessages"
also you do not need to start with Promise.resolve(), you can just start with the find().. chain 
and don't forget .catch(next) as all the other methods do

Comment (samparsky) [reply routes/channel.js]: done

File: services/validatorWorker/lib/propagation.js [hunk 2]
```diff
@@ -4,6 +4,11 @@ const db = require('../../../db')
 function propagate(adapter, receiver, channel, msg) {
 	return adapter.getAuthFor(receiver)
 	.then(function(authToken) {
+
```
Comment (Ivshti) [inline services/validatorWorker/lib/propagation.js]: should this be here?

Comment (samparsky) [reply services/validatorWorker/lib/propagation.js]: removed it

**Pull request merged**